test/parallel/test-repl-tab-complete.js

```diff
@@ -203,6 +203,86 @@ testMe.complete('toSt', common.mustCall(function(error, data) {
   assert.deepStrictEqual(data, [['toString'], 'toSt']);
 }));
 
+// tab completion Symbol
+{
+  assert.strictEqual(typeof repl.tabComplete, 'symbol');
+}
```

Comment:
This test does not need a own scope as it does not interfere with anything else.